json/src/spec/authority_round.rs

```diff
@@ -16,8 +16,9 @@
 
 //! Authority params deserialization.
 
-use ethereum_types::Address;
+use hash::Address;
```

Comment:
Previously it was not using the right type -- in `json`, we should use the json serialization/serialization type (`$crate::hash::Address`) rather than from `ethereum_types`.